jsapp/js/components/modalForms/projectSettings.es6

```diff
@@ -751,6 +751,10 @@ class ProjectSettings extends React.Component {
     const sectors = session.currentAccount.available_sectors;
     const countries = session.currentAccount.available_countries;
 
+    if (this.props.context === PROJECT_SETTINGS_CONTEXTS.EXISTING && this.props.iframeUrl) {
+      $('iframe#kc-settings-iframe').iFrameResize({log: true});
```

Comment:
the selector is unnecessarily specific.  Anyways, IDs are unique across applications.

Reply:
fixed :)